Jenkinsfile

```diff
@@ -1,119 +1,75 @@
 #!/usr/bin/env groovy
 
-@Library('sec_ci_libs@v2-latest') _
+@Library("sec_ci_libs@v2-latest") _
```

Comment:
It's not blocking for me, but could we have these style changes in a seperate commit next time? They are confusing to review

Reply:
You're right, but please consider my comment in the PR description, I created a new branch from master and copied over the Jenkinsfile - then deleted the release related parts.

Reply:
I think workflow issues shouldn't reflect on the commit history when it comes to a PR. But that might be my very own point of view on this. IMHO typing is not the bottlneck in this work here, so there could have been other approaches. Just pointing this out 😇